core/src/main/java/org/openedx/core/module/TranscriptManager.kt

```diff
@@ -67,9 +67,11 @@ class TranscriptManager(
             )
             if (result) {
                 getInputStream(downloadLink)?.let {
-                    val transcriptTimedTextObject =
-                        convertIntoTimedTextObject(it)
-                    transcriptObject = transcriptTimedTextObject
+                    try {
+                        transcriptObject = convertIntoTimedTextObject(it)
+                    } catch (e: Exception) {
```

Comment:
[Optional] UI/UX Improvement
Space underneath the video player if the app is unable to load the transcript.

![image](https://github.com/user-attachments/assets/549491f2-e6d3-4dae-bea4-85e3a6c3b4dd)

Reply:
As discussed, we will cover this in a separate PR.